sourcegraph/sourcegraph
# [Admin Analytics] add admin analytics usage stats ping

- Add usage stats ping for Admin Analytics page view.
- Fix font colors from black -> body.

## Test plan
- unit tests written for pings
- Visit https://sourcegraph.test:3443/site-admin/analytics/code-intel
- Turn on dark mode
- Check the legends colour to be white
